app/src/main/java/org/mozilla/focus/fragment/HomeFragment.kt

```diff
@@ -153,15 +196,26 @@ private class HomeTileAdapter(
             if (hasFocus) {
                 backgroundResource = R.drawable.home_tile_title_focused_background
                 textColor = Color.WHITE
+                menuButton.visibility = View.VISIBLE
             } else {
                 backgroundResource = 0
                 textColor = Color.BLACK
+                menuButton.visibility = View.GONE
             }
             titleView.setBackgroundResource(backgroundResource)
             titleView.setTextColor(textColor)
         }
     }
 
+    fun getItemAtPosition(position: Int): HomeTile {
+        return tiles[position]
+    }
+
+    fun removeItemAtPosition(position: Int) {
+        tiles.removeAt(position)
+        notifyItemRemoved(position)
```

Comment:
Again, should probably check if this position is allowed to be removed, if this is a public function. If you call notifyItemRemoved and the position is out of range, you'll get a crash.

Reply:
@bkmunar you didn't address this comment.